src/sdk/pynni/nni/compression/torch/speedup/compress_modules.py

```diff
@@ -141,13 +144,16 @@ def replace_conv2d(conv, mask):
     # channal is also divided into serveral groups and each group
     # filter may have different input channel indexes.
     input_step = int(conv.in_channels / conv.groups)
-    in_channels_group = int(in_channels / conv.groups)
-    filter_step = int(out_channels / conv.groups)
+    in_channels_group = int(in_channels / groups)
+    filter_step = int(conv.out_channels / conv.groups)
```

Comment:
What's the difference between the `groups` and `conv.groups`?

Reply:
conv.groups is the number of groups for original  conv layer.    groups is the  number of groups for new conv layer.    For depthwise conv layer,  the new groups is different than original one.